.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,26 @@
+name: Go test/lint
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
```

Comment:
I thought it would be better to specify the version instead of the `ubuntu-latest`, but it seems that there is no problem because it was the `ubuntu-latest` in the official document.
https://github.com/golangci/golangci-lint-action